cmd/kogito/command/install/data_index.go

```diff
@@ -164,17 +163,24 @@ func (i *installDataIndexCommand) Exec(cmd *cobra.Command, args []string) error
 		return nil
 	}
 
-	if i.flags.infinispan.UseKogitoInfra {
-		if available, err := infrastructure.IsInfinispanOperatorAvailable(i.Client, i.flags.Project); err != nil {
-			return err
-		} else if !available {
-			return fmt.Errorf("Infinispan Operator is not available in the Project: %s. Please make sure to install it before deploying Data Index without infinispan-url provided ", i.flags.Project)
+	// TODO this will be moved to the ServicesInstallerBuild API on KOGITO-911 PR
+	{
+		logger := context.GetDefaultLogger()
+		if i.flags.infinispan.UseKogitoInfra {
+			if infrastructure.IsInfinispanAvailable(i.Client) {
+				if available, err := infrastructure.IsInfinispanOperatorAvailable(i.Client, i.flags.Project); err != nil {
+					return err
+				} else if !available {
+					logger.Info(message.DataIndexInfinispanOperatorNotAvailable)
+				}
+			} else {
+				logger.Infof(message.DataIndexInfinispanNotAvailable, i.flags.Project)
+			}
 		}
-	}
-
-	if i.flags.kafka.UseKogitoInfra {
-		if available := infrastructure.IsStrimziAvailable(i.Client); !available {
-			return fmt.Errorf("Strimzi Operator is not available in the Project: %s. Please make sure to install it before deploying Data Index without Kafka provided ", i.flags.Project)
+		if i.flags.kafka.UseKogitoInfra {
+			if available := infrastructure.IsStrimziAvailable(i.Client); !available {
```

Comment:
Shouldn't we check for Strimzi operator as well ?

Reply:
We can't at this time (it's out of the scope of this JIRA). Also, their deployment is weird and the deployment hasn't a uniform name, we should put more thoughts/work on other way of doing that. Opened a JIRA to be tracked: https://issues.redhat.com/browse/KOGITO-2011